aibtc-v1/utils/crews.py

```diff
@@ -23,6 +24,7 @@ def create_crew(self) -> Crew:
             process=Process.sequential,
             verbose=True,
             memory=True,
+            embedder=st.session_state.embedder,
```

Comment:
We might want to pass this into the instance rather than tie the class to the st.session_state, same as we do with name.